### DISCUSSION TITLE
Need help getting NVENC HW encoding to work

### DISCUSSION THREAD

**Discussion by Saorn** [RandomNinjaAtk/docker-sabnzbd-extended#13] [General]
Need some direction on how to enable NVENC HE encoding with SMA in the docker.  I've passed the device to the docker and set the codec per your instructions but it's erroring out, I must be missing something.  Any help would be appreciated.  FYI I am using NVENC in a couple other dockers so I am familiar with how to pass the HW through to Plex and Tdarr.

**Comment by RandomNinjaAtk** [comment]
Hmm, probably legacy info sorry. But I believe when the base image moved from ubuntu base to alpine base, it became no longer supported. After checking the current image, it appears the ffmpeg build does not support Nvidia HW acceleration, VAAPI HW encoders should work though.

**Reply by Saorn** [reply]
Ah, oh well, I'm using an older gen Dell Poweredge server so my only HW encoder is the Quadro P2000 I added to it.  I was looking forward to be able to prencode everything rather doing all the post transcoding I do now on my library.  I'm loving theses new extended dockers you are creating btw, was planning to switch over to all of them to take advantage of your scripts :)

**Reply by RandomNinjaAtk** [reply]
If Intel Arc ever gets their act together, that would probably be the better solution, since nvidia can be problematic on linux at times....

I also pre-process using sabnzbd as my preferred method with SMA, however, I don't modify the quality, just drop languages, set lang preferences, and organize streams with it. So strictly a copy audio/video codec. I'd rather just pull in something that is closer to my desired output, then spending the time to re-encode.

**Reply by Saorn** [reply]
it's been pretty solid for me, I've literally done thousands of encodes at this point.
